src/components/Header/Header.tsx

```diff
@@ -12,8 +43,24 @@ export const Header = () => {
       <Navbar>
         <Navlink to={ROUTES_URL.HOME}>Home</Navlink>
         <Navlink to={ROUTES_URL.HOME}>About Us</Navlink>
-        <Navlink to={ROUTES_URL.HOME}>Register / Sign In</Navlink>
+        {isUserLoggedIn ? (
+          <SignOutLink onClick={signOutOnClick}>Sign Out</SignOutLink>
+        ) : (
+          <Navlink to={ROUTES_URL.AUTHENTICATION}>Register / Sign In</Navlink>
+        )}
       </Navbar>
+      {!userIsLoading && (
+        <NotificationModal
+          isOpen={isModalOpen}
+          status={userSignOutError ? "error" : "success"}
```

Comment:
you use these strings in other files too, can you please move them to a constant